src/plugins/intel_cpu/src/emitters/snippets/x64/jit_reg_spill_emitters.cpp

```diff
@@ -0,0 +1,90 @@
+// Copyright (C) 2020-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "jit_reg_spill_emitters.hpp"
+
+#include "emitters/plugin/x64/utils.hpp"
+#include "emitters/snippets/jit_snippets_call_args.hpp"
+#include "emitters/snippets/x64/utils.hpp"
+#include "snippets/utils/utils.hpp"
+
+using namespace Xbyak;
+using namespace dnnl::impl;
+using namespace dnnl::impl::cpu::x64;
+
+namespace ov {
+namespace intel_cpu {
+
+/* ================== jit_reg_spill_begin_emitters ====================== */
+
+jit_reg_spill_begin_emitter::jit_reg_spill_begin_emitter(dnnl::impl::cpu::x64::jit_generator* h,
+                                                         dnnl::impl::cpu::x64::cpu_isa_t isa,
+                                                         const ov::snippets::lowered::ExpressionPtr& expr)
+    : jit_emitter(h, isa),
+      m_reg_spill_begin_expr(expr) {
```

Comment:
Minor: do we need to store the whole expr? It seems like it's enough to store only `get_regs_to_spill()` result

Reply:
This was done because abstract registers weren't mapped to their physical counterparts at this point (since the mapping was performed by KernelEmitter). Now when we map directly in assign_registers we can do that.
Please note also that the spilled regs are copied to op::RegSpillBegin just to facilitate serialization and debugging, the primary place to store them is `reg_info`.